packages/client-sdk-nodejs/test/integration/shared/http/http-apis.test.ts

```diff
@@ -0,0 +1,7 @@
+import {SetupAuthClientIntegrationTest} from '../../integration-setup';
+import {runHttpApiTest} from '@gomomento/common-integration-tests/dist/src/http/http-apis';
+
+const {mgaAccountSessionTokenAuthClient, cacheName} =
+  SetupAuthClientIntegrationTest();
```

Comment:
Are these tests specific to mga account sessions?

Reply:
Not really, but the only way I could get the api-endpoint is by creating a new token using the mga account session token and then calculating the api endpoint from it.

Reply:
Is the MGA Account Session token a customer-facing concern? Ie can a customer run these integration tests?

Reply:
The customer won't be able to run these tests since they do not have any way to access mga's account session token.